jetty-websocket/javax-websocket-server/src/main/java/org/eclipse/jetty/websocket/javax/server/JavaxWebSocketServerContainer.java

```diff
@@ -283,20 +311,21 @@ public long getDefaultAsyncSendTimeout()
     public int getDefaultMaxBinaryMessageBufferSize()
     {
         // TODO: warn on long -> int conversion issue
-        return (int)this._webSocketCreatorMapping.getDefaultMaxBinaryMessageSize();
+        // TODO: Should this be Filter?
+        return (int)customizer.getMaxBinaryMessageSize();
     }
 
     @Override
     public long getDefaultMaxSessionIdleTimeout()
     {
-        return this._webSocketCreatorMapping.getDefaultIdleTimeout().toMillis();
+        return customizer.getIdleTimeout().toMillis();
     }
 
     @Override
     public int getDefaultMaxTextMessageBufferSize()
     {
         // TODO: warn on long -> int conversion issue
-        return (int)this._webSocketCreatorMapping.getDefaultMaxTextMessageSize();
+        return (int)customizer.getMaxTextMessageSize();
```

Comment:
Again, Careful of "Max_MessageBufferSize" and "Max_MessageSize"

Reply:
think this is correct - but thanks for the double check as it is confusing.